tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestSpecific.g.cs

```diff
@@ -1897,50 +1887,45 @@ public class PackageVersionsLatestSpecific
 #if DEFAULT_SAMPLES
                 new object[] { string.Empty },
 #else
-#if NET48
```

Comment:
👀

Reply:
done in [0db15ca](https://github.com/DataDog/dd-trace-dotnet/pull/7070/commits/0db15ca09c2ba13fb50db81c6ca7d527592886df)